plugins/processors/awsapplicationsignals/internal/resolver/attributesresolver.go

```diff
@@ -0,0 +1,160 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package resolver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	semconv1 "go.opentelemetry.io/collector/semconv/v1.17.0"
+	"go.uber.org/zap"
+
+	"github.com/aws/amazon-cloudwatch-agent/plugins/processors/awsapplicationsignals/common"
+	appsignalsconfig "github.com/aws/amazon-cloudwatch-agent/plugins/processors/awsapplicationsignals/config"
+	attr "github.com/aws/amazon-cloudwatch-agent/plugins/processors/awsapplicationsignals/internal/attributes"
+	"github.com/aws/amazon-cloudwatch-agent/translator/util/ecsutil"
+)
+
+const (
+	AttributeEnvironmentDefault = "default"
```

Comment:
should it be `generic:default` according https://quip-amazon.com/e3rkAe092WLA/GA-ApplicationSignals-EMF-Specification#temp:s:temp:C:ENV874364e7b79e48f79f78f3a68;temp:C:ENVc551808b05e64e9f8aff9865e

Reply:
nvm, found the composite value impl below